modules/priceFloors.js

```diff
@@ -291,6 +300,39 @@ export function updateAdUnitsForAuction(adUnits, floorData, auctionId) {
         location: utils.deepAccess(floorData, 'data.location', 'noData'),
         floorProvider: floorData.floorProvider,
         fetchStatus: _floorsConfig.fetchStatus
+      };
+
+      if (utils.deepAccess(floorData, 'floorMin')) {
+        bid.floorData.floorMin = utils.deepAccess(floorData, 'floorMin');
+      }
+    });
+  });
+}
+
+/**
+ * @summary This function takes the adUnits for the auction and update them accordingly as well as returns the rules hashmap for the auction
+ */
+export function updateAdUnitFloorData(adUnits, floorData, auctionId) {
+  adUnits.forEach((adUnit) => {
+    adUnit.bids.forEach(bid => {
+      if (floorData.skipped) {
+        delete bid.getFloor;
+      } else {
+        bid.getFloor = getFloor;
+      }
+      // information for bid and analytics adapters
+      bid.auctionId = auctionId;
+      bid.floorData = {
+        skipped: floorData.skipped,
+        skipRate: floorData.skipRate,
+        modelVersion: utils.deepAccess(floorData, 'data.modelVersion'),
+        location: utils.deepAccess(floorData, 'data.location', 'noData'),
+        floorProvider: floorData.floorProvider,
+        fetchStatus: _floorsConfig.fetchStatus
+      };
+
+      if (utils.deepAccess(floorData, 'floorMin')) {
+        bid.floorData.floorMin = utils.deepAccess(floorData, 'floorMin');
```

Comment:
Think something went weird here during commit.

I do not see this function being called at all `updateAdUnitFloorData`

I think the changes needed were made in `updateAdUnitsForAuction` but somehow duplicated and function name changed.

So I think you can remove this entire selection.